scripts/resolve_network.py

```diff
@@ -70,43 +88,45 @@ def add_H2(n, snakemake):
     #        p_min_pu=-1)
 
     n.add("Load",
-          f"{name} H2",
-          carrier=f"{name} H2",
```

Comment:
I am not sure about the role of the carrier keyword. I found it strange that it included "name", so I deleted it. Now I was thinking this may cause the error, but including "name" in the carrier keyword does not seem to solve the problem on my end...